controllers/tab/publicationEntry/PublicationEntryTabHandler.inc.php

```diff
@@ -106,10 +106,15 @@ function authorize($request, &$args, $roleAssignments) {
 	 * @return JSONMessage JSON object
 	 */
 	function submissionMetadata($args, $request) {
+		$revision = $request->getUserVar('revision') ? (int)$request->getUserVar('revision') : null;
 
-		$publicationEntrySubmissionReviewForm = $this->_getPublicationEntrySubmissionReviewForm();
+		if(isset($args['submissionRevision'])){
+			$revision = $args['submissionRevision'];
+		}
```

Comment:
hmmm... Are those two things different: `$request->getUserVar('revision')` and `$args['submissionRevision']`?